ts/features/payments/details/components/PaymentsMethodPspDetailsBottomSheet.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { VSpacer, Body } from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import { useIOBottomSheetAutoresizableModal } from "../../../../utils/hooks/bottomSheet";
+
+type PaymentsMethodPspDetailsBottomSheetProps = {
+  pspBusinessName: string;
+};
+
+/**
+ * this component shows information about how does it works a psp preselected
+ * @constructor
+ */
+const PaymentsMethodPspDetailsBottomSheet = ({
+  pspBusinessName
+}: PaymentsMethodPspDetailsBottomSheetProps) => (
+  <>
+    <Body>
+      {I18n.t("features.payments.details.pspAlert.explainationContent", {
+        pspBusinessName
+      })}
+    </Body>
+    <VSpacer size={16} />
```

Comment:
Did you try to use the `bottomPadding` parameter in the `useIOBottomSheetAutoresizableModal` hook?

Reply:
Yes and the behavior is the same, what changes?

Reply:
If we need some extra padding we should use the `bottomPadding` parameter.  I think it's a more robust implementation.

Reply:
I am not sure about the robustness of the logic for adding the bottom padding through using that parameter. In fact, by passing the same `16` value as is also shown in the figma for the space needed from below, the bottomSheet padding seems to be broken, as if it does not take into account the safe area view.

<img src="https://github.com/pagopa/io-app/assets/34343582/31d9f96a-2961-446e-8665-cc855fbadfd2" width="300" />


That said, seems that using the `VSpacer` is the best solution right now.

Reply:
> I am not sure about the robustness of the logic for adding the bottom padding through using that parameter. In fact, by passing the same `16` value as is also shown in the figma for the space needed from below, the bottomSheet padding seems to be broken, as if it does not take into account the safe area view.
> 
> <img alt="" width="300" src="https://private-user-images.githubusercontent.com/34343582/338725700-31d9f96a-2961-446e-8665-cc855fbadfd2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNzczMjQsIm5iZiI6MTcxODE3NzAyNCwicGF0aCI6Ii8zNDM0MzU4Mi8zMzg3MjU3MDAtMzFkOWY5NmEtMjk2MS00NDZlLTg2NjUtY2M4NTVmYmFkZmQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEyVDA3MjM0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1NGFjYWEyZWFlZGZkMDc2Mzc0OTQ5YWIxZDY3ZThiNGE1ZTcyNzUwYjBhYjJiNTY3NDY3NmMyYzQ3MTg4ZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OE1h5BmgbeFRQGrnSAC6UnCbY5pfl7c42BvbV2xX5nc">
> That said, seems that using the `VSpacer` is the best solution right now.

Yes, 16 is not enough. Take a look at the hook implementation: the default bottomPadding is 72. So if you pass a <VSpacer size={16} />, you end up with an 88 bottom padding.

What happens if, one day, the library is updated and the spacing is no longer required? By using the parameter, we can easily remove it from everywhere without changing a lot of code. With the `VSpacer`, we would have to check and fix every bottom padding individually. What do you think about this possibility?

Reply:
If we want to consider a pixel-perfect design, we cannot go by eye so that the padding value passed as an argument is equal to the value given in Figma. From the design it is indicated that we need `16 pixels` of space from the bottom, however to achieve the same result we must pass `88` as the value of the bottomSheet argument. I agree with what you are saying but as it is currently implemented that hook, I think that is "broken" since it doesn't set the correct number of pixels starting from the bottom (I mean, what if in the feature other devices require more space from the bottom? Actually that hook is not versatile and doesn't take into account the safe area bottom)

Reply:
> If we want to consider a pixel-perfect design, we cannot go by eye so that the padding value passed as an argument is equal to the value given in Figma. From the design it is indicated that we need `16 pixels` of space from the bottom, however to achieve the same result we must pass `88` as the value of the bottomSheet argument. I agree with what you are saying but as it is currently implemented that hook, I think that is "broken" since it doesn't set the correct number of pixels starting from the bottom (I mean, what if in the feature other devices require more space from the bottom? Actually that hook is not versatile and doesn't take into account the safe area bottom)

Yes you are right, but remember that the `VSpacer` is working because there is a bottom padding applied inside the hook. 
Let's use it for now.